.github/workflows/stable.yml

```diff
@@ -199,11 +202,11 @@ jobs:
 
       - name: Run cargo build
         if: endsWith(matrix.target, '-gnu')
-        run: cargo build --target=${{ matrix.target }} --verbose --all-targets ${{ env.DEFAULT_OPTIONS }} --features=boringssl-boring-crate
+        run: cargo build --target=${{ matrix.target }} --verbose --all-targets ${{ env.DEFAULT_OPTIONS }} --workspace --exclude qlog-dancer --features=boringssl-boring-crate
```

Comment:
I think you could potentially just move `--workspace` into the `DEFAULT_OPTIONS` variable instead so you don't need to duplicate it below.